code/workspace/linting-and-testing/example_vpc_and_security_group.yaml

```diff
@@ -0,0 +1,34 @@
+AWSTemplateFormatVersion: "2010-09-09"
+
+Description: AWS CloudFormation workshop - linting and testing - example VPC and Security Group (uksb-1q9p31idr)
```

Comment:
```suggestion
Description: AWS CloudFormation workshop - Linting and testing - VPC and Security Group template (uksb-1q9p31idr)
```

Reply:
Please rename the file to `vpc-and-security-group.yaml`. We should avoid naming files and variables with words such as `example`, `my` etc.

Reply:
This file is missing in `code/solutions/` directory, please add the final working file there.